vm/package/build.rs

```diff
@@ -231,7 +228,7 @@ impl<N: Network> Package<N> {
                         CallOperator::Locator(locator) => {
                             (process.get_program(locator.program_id())?, locator.resource())
                         }
-                        CallOperator::Resource(resource) => (program, resource),
+                        CallOperator::Resource(resource) => (program.clone(), resource),
```

Comment:
```suggestion
                        CallOperator::Resource(resource) => (program, resource),
```
`get_program` no longer returns a reference

Reply:
ah, but it's used down the line :+1: